src/test/java/stepDefinition/MakeAppointment.java

```diff
@@ -0,0 +1,101 @@
+package stepDefinition;
+
+import cucumber.api.PendingException;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.junit.Assert;
+import pageObjects.login.AppointmentConfirmPage;
+import pageObjects.login.AppointmentPage;
+import pageObjects.login.Homepage;
+import pageObjects.login.LoginPage;
+import utility.GetPropertiesValue;
+import utility.TestBase;
+
+public class MakeAppointment extends TestBase {
+
+    private GetPropertiesValue getPropertiesValue = new GetPropertiesValue();
+
+    @Given("^user signed in with \"([^\"]*)\" and \"([^\"]*)\"$")
+    public void userSignedWithAnd(String arg1, String arg2) {
+        driver.get(getPropertiesValue.getPropValues("Base_URL.properties", "BASE_URL") + "/profile.php#login");
+        LoginPage.username().sendKeys("John Doe");
+        LoginPage.password().sendKeys("ThisIsNotAPassword");
+        LoginPage.loginButton().click();
+    }
```

Comment:
xóa step này đi, đã có step này `user signed in with acc \"([^\"]*)\" and \"([^\"]*)\"$`, còn viết thêm step mới làm gì?